src/plugins/opensearch_dashboards_utils/docs/global_query_param.md

```diff
@@ -0,0 +1,99 @@
+# Global query parameters
+
+Current there are five plugins that have implemented global data persistence ability in Opensearch Dashboards, and they are visualize, discover, timeline, dashboards, and vis-builder. Global query parameters include globally pinned filters, time range and time refresh intervals. These parameters are not only persisted across the action of refresh, but also persisted among all the plugins that have global data persistence ability. 
```

Comment:
* Do we want to date this? Like `As of 12/1/2022, there are` I tend to prefer this method as opposed to `Current` because if neglect to update this then the reader can understand that at the time of writing this vs pulling it down in the future
* `Opensearch Dashboards` --> `OpenSearch Dashboards`
* `timeline` is technically is the "branding" of the plugin so I think it should be `Timeline`

Do we want to mention that these are plugins **within** OpenSearch Dashboards "core". Or do we know for a fact that external plugins don't hook into it.

Reply:
Thanks! Will replace current with the current date, and I am not sure if external plugins hook into this or not, I will mention that these are only the plugins within OpenSearch Dashboards core.